pkg/wallet/models.go

```diff
@@ -30,6 +30,7 @@ type QueryParams struct {
 // To be used as options for issue/prove wallet features.
 //
 type ProofOptions struct {
+	KID string `json:"kid,omitempty"`
```

Comment:
I am ok to pass this as an option for generating proofs, but it shouldn't get replaced internally by wallet. That will break the interoperability.